decisions/0002-record-architecture-decisions.md

```diff
@@ -0,0 +1,49 @@
+
+## Consequences
+
+In the past, NIST staff have sent individual calendar invites from their individual accounts' calendars to specific individuals or the oscal-dev@list.nist.gov mailing list at large. Additionally, the team had published individual iCalendar (ICS) files for two of the long-standing public meetings. The latter had not been updated frequent and were out of sync until removed completely with [pull request usnistgov/OSCAL#1614](https://github.com/usnistgov/OSCAL/pull/1614). The former approach, although convenient, presents a few challenges.
+
+1. It is difficult to quickly access one or more event invitations after they are sent.
+1. Only one staff member in the team, the event's creator, can manage that event's invitation details, often when multiple staff are involved.
+1. Community members often accept an invitation email via the mailing list once or several times, but not every time. They lose important updates to event details and contact us for clarification.
+
+Therefore we need a solution that meets the following requirements for the use case of notifying community members to public meetings.
+
+1. A system for all members of the NIST OSCAL Team to create
+1. A system for all members of the NIST OSCAL Team to modify and remove existing event invitations
+1. A system that allows any anonymous community member, without prior or ongoing authentication and authorization to a NIST-hosted service, to access event information
+1. A system that has a standards conformant, or largely popular, calendar synchronization mechanism for authorized NIST staff and anonymous community members (iCalendar or otherwise)
+1. Manage frequent changes to calendar events efficiently with short notice without waiting upon approval times by team, group, division, or lab leadership
+
+With these high-level requirements in mind, we have several options.
+
+## CMS for the NIST Computer Security Resource Center Website and Calendar
+
+As part of a spike, we evaluated event management [via the content management system that published the csrc.nist.gov website, specifically the Events page](https://csrc.nist.gov/Events). Although this solution partially supports requirements 3 and 4, it does not completely support requirement 1 or 4 at all. The CMS has a permissions system that only works with one group within the respective division for those who manage the content. At this time, the NIST OSCAL Team is composed of staff from teams from two different group's in ITL's Computer Security Division, so requirements 1 and 2 are not easily achievable or unsupported based upon consultation with the web developers that maintain the system. Additionally per requirement 5, there will need to be approval for every change, imposing upon approvers for events where we have historically made frequent changes.
```

Comment:
Should the sentence above read "Although this solution partially supports requirements 3 and 4, it does not completely support requirement 1 or 2 at all." (not 4)?

Reply:
```suggestion
As part of a spike, we evaluated event management [via the content management system that published the csrc.nist.gov website, specifically the Events page](https://csrc.nist.gov/Events). Although this solution partially supports requirements 3 and 4, it does not completely support requirement 1 or 2 at all. The CMS has a permissions system that only works with one group within the respective division for those who manage the content. At this time, the NIST OSCAL Team is composed of staff from teams from two different group's in ITL's Computer Security Division, so requirements 1 and 2 are not easily achievable or unsupported based upon consultation with the web developers that maintain the system. Additionally per requirement 5, there will need to be approval for every change, imposing upon approvers for events where we have historically made frequent changes.
```